tests/unit_tests/test_mesh_from_domain.py

```diff
@@ -83,7 +83,7 @@ def test_cylindrical_mesh_from_region():
     assert np.array_equal(mesh.dimension, (6, 2, 3))
     assert np.array_equal(mesh.r_grid, [0., 1., 2., 3., 4., 5., 6.])
     assert np.array_equal(mesh.phi_grid, [0., 0.5*np.pi, np.pi])
-    assert np.array_equal(mesh.z_grid, [-30., -10., 10., 30.])
+    assert np.array_equal(mesh.z_grid, [0.0, 20., 40., 60])
```

Comment:
would it be possible to add in an origin assert here as well, it helps me check that the absolute location of the z elements are correct as I am keen to add up z_grid with origin just to see if it works out

```assert np.array_equal(mesh.origin,...```

Reply:
good call. Done!